AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -109,11 +110,14 @@ extension BluetoothManager: CBCentralManagerDelegate {
         // Here's code for getting data from AB.
         peripheral.discoverServices(nil)
         peripheral.delegate = self
+        connectedPeripheral = peripheral
     }
     
     func centralManager(_ central: CBCentralManager, didDisconnectPeripheral peripheral: CBPeripheral, error: Error?) {
         print("Disconnected: \(String(describing: error?.localizedDescription))")
-        mobilePeripheralSessionManager.finishSession(for: peripheral)
+        connectedPeripheral = nil
+        mobilePeripheralSessionManager.finishSession(for: peripheral,
```

Comment:
I really don't like that we're ending session as a side effect of a callback function inside BluetoothManager :D This makes the code structure and getting where things happen really hard